ipatests/test_integration/test_commands.py

```diff
@@ -577,3 +578,45 @@ def test_config_show_configured_services(self):
             # reset
             entry['ipaConfigString'] = orig_cfg
             conn.update_entry(entry)  # pylint: disable=no-member
+
+    def test_ssh_from_controller(self):
```

Comment:
The fix is available in sssd 2.2.0, which is provided in updates for fedora 30 but is currently in updates-testing in fedora29. IIRC updates-testing is not enabled in the VM images, which means that the test is likely to fail for testing_f29 nightlies.
You should add a xfail related to sssd version on the master.

Reply:
added xfail for the test